backend/routes/routes.js

```diff
@@ -59,7 +59,7 @@ router.post('/approve', approveController.createApprove);
 router.delete('/approve/:approve_id', approveController.deleteApprove);
 router.get('/approve/:approve_id', approveController.getApproveInfo);
```

Comment:
```suggestion
router.get('/approve/detail/:approve_id', approveController.getApproveInfo);
```

이게 user_id만 없애서 괜찮은 문제인지 걱정이 됩니닷.
저번과 같은 오류일 수 있으니,
만일을 위해 `/approve/:approve_id`와 `approve/list/:ch_id` 구분을 두는 건 어떨까요?

Reply:
좋은 생각인 것 같습니다. 원래 있던 api도 복구해놓을게요!

Reply:
제가 체크한 부부는 구분이 안되어있네욥!